src/utils.ts

```diff
@@ -0,0 +1,114 @@
+import Fetcher from './fetcher'
```

Comment:
Am I correct in understanding that `util` contains utility methods written in Javascript, and `utils` methods written in TypeScript? If so, I'm not sure that pluralising it is a good way of discerning between the two - maybe just rename `util.js` to something like `jsUtils.js`, for example?

Reply:
Agreed. Renamed to utils-js.js (so they appear next to eachother alphabetically)